llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -117,14 +117,21 @@ struct PortableMemInfoBlock {
   void clear() { *this = PortableMemInfoBlock(); }
 
   // Returns the full schema currently in use.
-  static MemProfSchema getSchema() {
+  static MemProfSchema getFullSchema() {
     MemProfSchema List;
 #define MIBEntryDef(NameTag, Name, Type) List.push_back(Meta::Name);
 #include "llvm/ProfileData/MIBEntryDef.inc"
 #undef MIBEntryDef
     return List;
   }
 
+  // Returns the schema consisting of the fields currently consumed by the
+  // compiler.
+  static MemProfSchema getSchema() {
```

Comment:
Maybe we should qualify the schema based on the use case. For example these fields are necessary for hot-cold annotation, so name this `getHotColdSchema()`?

Reply:
I'll incorporate the suggested name in this patch.  Thanks!